src/chainstate/stacks/db/accounts.rs

```diff
@@ -436,17 +424,17 @@ impl StacksChainState {
         Ok(rows)
     }
 
-    /// Get all scheduled miner rewards for a given index hash
-    pub fn get_scheduled_block_rewards(
-        conn: &DBConn,
-        index_block_hash: &StacksBlockId,
+    /// Get the scheduled miner rewards in a particular Stacks fork at a particular height.
+    pub fn get_scheduled_block_rewards<'a>(
+        tx: &mut StacksDBTx<'a>,
+        tip: &StacksHeaderInfo,
     ) -> Result<Vec<MinerPaymentSchedule>, Error> {
-        let qry =
-            "SELECT * FROM payments WHERE index_block_hash = ?1 ORDER BY vtxindex ASC".to_string();
-        let args: &[&dyn ToSql] = &[index_block_hash];
-        let rows =
-            query_rows::<MinerPaymentSchedule, _>(conn, &qry, args).map_err(Error::DBError)?;
-        Ok(rows)
+        if tip.block_height < MINER_REWARD_MATURITY {
```

Comment:
`<=` ? Not sure, but I don't think that Block 0 is a rewarding block

Reply:
It's `<` because of the code below:

```
let block_height = tip.block_height - MINER_REWARD_MATURITY;
```

`block_height` needs to be at least 1 -- there's no block reward at height 0, since that's the boot block.